app/Models/Facility.php

```diff
@@ -43,17 +44,14 @@ class Facility extends Model
     protected static function fieldValidator(string $field): string|array
     {
         return match ($field) {
-            'name' => 'required|string',
-            'url' => [
-                'bail',
-                'required',
-                'string',
-                'max:15',
+            'name' => Valid::trimmed(),
+            'url' => Valid::trimmed([
+                'max:30',
                 'lowercase',
                 'regex:/^[a-z][a-z0-9-]+[a-z0-9]$/',
```

Comment:
Jeżeli to jest faktycznie URL placówki, to ten regex powinien wyglądać jakoś tak:
`[(http(s)?):\/\/(www\.)?a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)`
src: https://regexr.com/39nr7

Reply:
Jak już, to 'url:http,https', laravel ma takie rzeczy wbudowane p;